src/main/kotlin/com/strumenta/kolasu/model/Processing.kt

```diff
@@ -178,7 +144,7 @@ fun Node.processConsideringParent(operation: (Node, Node?) -> Unit, parent: Node
 }
 
 /**
- * @return all descendants of this node, meaning all the children, the children of those children, etc.
+ * @return all direct children of this node.
```

Comment:
So here the doc was stating a lie!

Reply:
Too bad it was not caught in the review ;-)